assets/react/components/grid-item.jsx

```diff
@@ -54,9 +96,38 @@ const ItemIcon = styled.img`
   height: 20%;
 `;
 
-const Item = styled.div`
+const Skeleton = styled.div`
+  @keyframes gradient {
+    0% {
+      background-position: 0% 50%;
+    }
+    50% {
+      background-position: 100% 50%;
+    }
+    100% {
+      background-position: 0% 50%;
+    }
+  }
+
+  width: calc(100vw / var(--grid-columns));
+  height: calc(100vh / var(--total-rows));
+  background: linear-gradient(
+    -45deg,
+    var(--skeleton-color-3),
+    var(--skeleton-color-2),
+    var(--skeleton-color-1)
+  );
+  background-size: 400% 400%;
+  animation: gradient 2s ease infinite;
+`;
+
+const ContentWrapper = styled.div`
+  position: relative;
+`;
+
+const Item = styled.img`
+  object-fit: cover;
   position: relative;
-  background-image: var(--background-image);
   background-position: center;
```

Comment:
Are bg image and position still relevant?

Reply:
prob not, will remove